sdk/search/search-documents/review/search-documents-node.api.md

```diff
@@ -260,6 +333,7 @@ export interface BaseSearchRequestOptions<TModel extends object, TFields extends
     speller?: QuerySpeller;
     top?: number;
     vectorSearchOptions?: VectorSearchOptions<TModel>;
+    xMsQuerySourceAuthorization?: string;
```

Comment:
what is this used for? Is it needed for customers of this client library?

Reply:
Per offline discussion, this is intended.